randovania/games/samus_returns/gui/ui_files/msr_cosmetic_patches_dialog.ui

```diff
@@ -46,10 +46,17 @@
         <x>0</x>
```

Comment:
screenshot for this file pls

Reply:
![image](https://github.com/randovania/randovania/assets/38679103/aa657657-68b7-48be-950b-7a0215077c6c)

Reply:
Updated the window again to mention that it does not work with Luma3DS

![image](https://github.com/randovania/randovania/assets/38679103/f01021d5-5e83-429f-9a54-b9025075176a)